### DISCUSSION TITLE
Epic Content Stack

### DISCUSSION THREAD

**Discussion by arpitdalal** [epicweb-dev/epic-stack#709] [Show and tell]
I reach for Epic Stack for ease of setup and getting up and running quickly, be it testing or deployment. I get a lot of best practices and opinions out of the box. But as minimal as it is, it still has a lot of things that might not be suitable for certain use cases.

One of them is using Epic Stack for content sites like landing pages, portfolios, or even building custom documentation sites.

I personally have reached Epic Stack for this exact purpose twice now and every time I had to strip down Prisma, SQLite, LRU cache, E2E tests, and authentication code. Because of that, I had to change how to deploy to Fly and modify GH actions too. Removing code and dependencies consumed valuable time that could have been spent on the app logic.

So, I created [epic-content-stack](https://github.com/arpitdalal/epic-content-stack) for this purpose which offers a great starting point for any content-heavy site, that also embraces all the best practices and opinions of the Epic Stack.

**Comment by kentcdodds** [comment]
This is so awesome! Lots of folks have asked for this so I'm glad you put it together 😄 thanks!
